File: src/sdk-reference/index/exist/snippets/exists.js
kuzzle
  .index
  .exists('nyc-open-data')
  .then(exists => {
    if (exists === true) {
      console.log('index exists')
    } else {
      console.log('index does not exist')
    }
  })
  .catch(error => {
    console.error(error.message)
  });

File: src/sdk-reference/cpp/1/collection/exists/snippets/exists.cpp
try {
  bool exists = kuzzle->collection->exists("nyc-open-data", "green-taxi");

  if (exists) {
    std::cout << "Collection green-taxi exists" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
